src/azure-cli/azure/cli/command_modules/vm/aaz/latest/sig/gallery_application/version/_delete.py

```diff
@@ -45,13 +45,13 @@ def _build_arguments_schema(cls, *args, **kwargs):
         _args_schema = cls._args_schema
         _args_schema.gallery_application_name = AAZStrArg(
             options=["--application-name", "--gallery-application-name"],
-            help="The name of the gallery application.",
+            help="The name of the gallery Application.",
```

Comment:
It is better to keep the help message unchanged, since `gallery application` seems to be more sensible.

Reply:
Please check other occurrences as well.

Reply:
Updated help summaries.